socorro/scripts/pubsub.py

```diff
@@ -0,0 +1,184 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Pub/Sub manipulation script.
+
+import argparse
+import os
+
+from google.api_core.exceptions import AlreadyExists, NotFound
+from google.cloud import pubsub_v1
+
+from socorro.scripts import WrappedTextHelpFormatter
+
+
+DESCRIPTION = 'Local dev environment Pub/Sub emulator manipulation script.'
+
+ENV_VARS = [
+    'resource.pubsub.project_id',
+    'resource.pubsub.standard_topic_name',
+    'resource.pubsub.standard_subscription_name',
+    'resource.pubsub.priority_topic_name',
+    'resource.pubsub.priority_subscription_name',
+    'resource.pubsub.reprocessing_topic_name',
+    'resource.pubsub.reprocessing_subscription_name'
+]
+
+EPILOG = 'Requires %s, and %s to be set in the environment.' % (
+    ', '.join(ENV_VARS[:-1]), ENV_VARS[-1]
+)
+
+# Number of seconds Pub/Sub should wait for a message to be acknowledged
+ACK_DEADLINE = 120
```

Comment:
I think 120 seconds works fine for local development. We can use a different value in stage and prod.